montecarlo/ssa.py

```diff
@@ -75,11 +75,17 @@ def __init__(
         initial_conditions,
         propensities,
         stoichiometry,
+        total_steps=None
         max_duration=None
     ):
         """Initialize model"""
         super().__init__(**initial_conditions)
+        for i, dictionaries in enumerate(stoichiometry):
```

Comment:
`__init__` feels like the best member to contain reserved-word logic to me, but maybe we can raise an exception instead?    

```python
for sto in stoichiometry:
    if "time" in sto:
        raise Exception("error, 'time' can't be in stoichiometry")
```

Reply:
Yes better!